environment.yml

```diff
@@ -32,7 +33,7 @@ dependencies:
   - netcdf4
   - numpy
   - palettable
-  - pandas
+  - pandas<1.1
```

Comment:
work around https://github.com/pydata/xarray/issues/4283

Reply:
This won't affect the hub at the moment but I'll propagate the change there tomorrow.

Do you want us to merge this or are you firing up binder from your fork?

Reply:
This won't affect the hub at the moment but I'll propagate the change there tomorrow.

Do you want us to merge this or are you firing up binder from your fork?

Reply:
> or are you firing up binder from your fork?

Don;t know how to do this!

Let's merge for now.

Reply:
> Don't know how to do this!

You can create your URL with:

`https://binder.pangeo.io/v2/gh/<yourusername>/ohw20-tutorials/<your-branch>?urlpath=lab`